mkinitramfs-ll.sh

```diff
@@ -25,6 +25,7 @@ usage() {
   -c, --compressor='gzip -9'  Use 'gzip -9' compressor instead of default
```

Comment:
I don't see any value with this addition... here grub detect all initramfs files ending with cpio.{gz,xz...}. So, this won't be merged.